src/main/resources/script.sh

```diff
@@ -0,0 +1,1218 @@
+#!/bin/bash
+
+function info {
+  local D=`date`
+  echo [ INFO - $D ] $*
+}
+
+function warning {
+  local D=`date`
+  echo [ WARN - $D ] $*
+}
+
+function error {
+  local D=`date`
+  echo [ ERROR - $D ] $* >&2
+}
+
+function startLog {
+  echo "<$*>" >&1
+  echo "<$*>" >&2
+}
+
+function stopLog {
+  local logName=$1
+  echo "</${logName}>" >&1
+  echo "</${logName}>" >&2
+}
+
+# Extract filename without extension
+DIRNAME=$(basename "${0%.sh}")
+
+# Check if directories already exist (In case of LOCAL, the directories already exists. To replicate the LOCAL execution in DIRAC, we create the directories on the remote node)
+if [[ ! -d "config" || ! -d "inv" ]]; then
+    # Create the directories if they don't already exist
+    mkdir -p inv
+    mkdir -p config
+
+    # Copy the files to their respective directories after creation
+    cp "${filename}-configuration.sh" config/
+    echo "Copied ${DIRNAME}-configuration.sh to config/"
+    cp "${filename}-invocation.json" inv/
```

Comment:
why not use $invocationJson (rename it to `invocationJsonFilename`), and do the same for the config (like `configurationFilename`